argo-cd-apps/base/smee-client/smee-client.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     spec:
       project: default
       source:
-        path: '{{values.sourceRoot}}'
+        path: '{{values.sourceRoot}}/{{values.environment}}/{{values.clusterDir}}'
```

Comment:
Why do we need those directories in the path string if they don't actually exist in the repo?

Reply:
they exist